Bottom: e592c3a66b0011a076c45be89dc0e7ddb992d654
Top:    d3e59cede44c8b8296ca26dc2c5c941d3aa3b3ae
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-05 08:57:48 -0500

Refresh of buildbot-playbook

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index c7e10ae1..9827d861 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -101,7 +101,7 @@
     - buildsystem_container
 
 - name: Setup the buildbot containers
-  include: buildbot.yml
+  import_playbook: buildbot.yml
 
 - name: Setup the Build System (Test Station) with repo data
   hosts: buildsystem
